client/ayon_core/pipeline/farm/pyblish_functions.py

```diff
@@ -708,6 +833,36 @@ def _create_instances_for_aov(instance, skeleton, aov_filter, additional_data,
     return instances
 
 
+def _collect_expected_files_for_aov(files):
+    """Collect expected files.
+
+    Args:
+        files (list): List of files.
+
+    Returns:
+        list or str: Collection of files or single file.
+
+    Raises:
+        ValueError: If there are multiple collections.
+
+    """
+    cols, rem = clique.assemble(files)
+    # we shouldn't have any reminders. And if we do, it should
+    # be just one item for single frame renders.
+    if not cols and rem:
+        if len(rem) != 1:
+            raise ValueError("Found multiple non related files "
+                             "to render, don't know what to do "
+                             "with them.")
+        return rem[0]
+    else:
+        # but we really expect only one collection.
+        # Nothing else make sense.
+        if len(cols) != 1:
+            raise ValueError("Only one image sequence type is expected.")  # noqa: E501
+        return list(cols[0])
```

Comment:
Unnecesasry else statement.
```suggestion
    # but we really expect only one collection.
    # Nothing else make sense.
    if len(cols) != 1:
        raise ValueError("Only one image sequence type is expected.")  # noqa: E501
    return list(cols[0])
```